tests/e2e/tests/regression/browser/database-overview.e2e.ts

```diff
@@ -38,3 +40,13 @@ test
         await t.expect(workbenchPage.overviewTotalKeys.exists).ok('User can see total keys');
         await t.expect(workbenchPage.overviewTotalMemory.exists).ok('User can see total memory');
     });
+test
+    .meta({ rte: rte.standalone })
+    .after(async () => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+    ('Verify that user can connect to DB and see breadcrumbs at the top of the application in Browser page', async t => {
+        //Verify that user can see breadcrumbs
+        await t.expect(browserPage.breadcrumbsContainer.visible).ok('User can see breadcrumbs in Browser page', { timeout: 20000 });
```

Comment:
1 line test? couldn't it be covered over lower levels (such as components)

Reply:
another verification is added in this test